src/tests/redux/reducers/util.test.js

```diff
@@ -0,0 +1,21 @@
+import { filterClass, filterList, filterSources } from '../../../redux/reducers/util';
+
+const arg = 'arg';
+
+describe('filterSources', () => {
+  it('should return an empty list if its argument is not an array', () => {
```

Comment:
Does this mean the argument can be anything?
Do we have a test for when the argument is as expected?

Reply:
> Do we have a test for when the argument is as expected?

I had assumed so, since the lines were being covered, but looking through the code base, it appears that may not be the case.
Adding these tests.